src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -230,14 +243,17 @@ private String renderLiveViewSql(
     liveViewSql.add("transactIdCol", PDAO_TRANSACTION_ID_COLUMN);
     liveViewSql.add("transactStatusCol", PDAO_TRANSACTION_STATUS_COLUMN);
     liveViewSql.add("partitionDateCol", PDAO_INGEST_DATE_COLUMN_ALIAS);
+    liveViewSql.add("transactionTerminatedAtCol", PDAO_TRANSACTION_TERMINATED_AT_COLUMN);
     liveViewSql.add("transactStatusVal", TransactionModel.StatusEnum.ACTIVE);
 
     liveViewSql.add("columns", PDAO_ROW_ID_COLUMN);
     liveViewSql.add(
         "columns", table.getColumns().stream().map(Column::getName).collect(Collectors.toList()));
     liveViewSql.add(
         "partitionByDate",
-        table.getBigQueryPartitionConfig().getMode() == BigQueryPartitionConfigV1.Mode.INGEST_DATE);
+        table.getBigQueryPartitionConfig() != null
```

Comment:
It looks like there's a number of places where our code assumes that `table.getBigQueryPartitionConfig()` will be non-null. Is there a reason why a check is needed here but not elsewhere, such as `BigQueryPDao.createDataset()`? Would it be possible to have a default config set so a null check wouldn't be necessary here?

Reply:
Hmm...I'm actually seeing that there's a `BigQueryPartitionConfigV1.NONE` that should be used instead of null.  I ran into an npe here...lemme see if I can reproduce

Reply:
I might leave this (since I'd prefer to be overly defensive)